README.md

```diff
@@ -22,13 +27,47 @@ Directory `src` contains the following files:
 - `-b` - Bit size of the secrets (1-64). Default value 32.
 - `-s` - Number of shares. Default value 1.
 - `-t` - Number of traces in one experiment. Default value 100K.
-- `-n` - Amplitude of normally distributed noise added to the traces. By default no noise (= amplitude 0).
+- `-n` - Amplitude of normally distributed noise added to the traces. Default value 0 (no noise).
 - `-e` - Number of experiments. Default value 1.
-- `-r` - Random seed (by default `None`). If no random seed is provided, the experiments are not reproducible, since each time different random values are used. If a random seed is provided, the experiments are reproducible, and the same command line always produces the same result.
+- `-r` - Random seed. If no random seed is provided, the experiments are not reproducible, since each time different random values are used. If a random seed is provided, the experiments are reproducible, and the same command line always produces the same result.
 - `-v` - Verbose. Permissible only if the number of experiments is 1 (which is the default). Outputs a summary of the rounds corresponding to bits 0-7 (or less if the bit size is less than 8).
 - `-l` - Long printout. Permissible only if the number of experiments is 1 (which is the default). Outputs the same summary as with `-v` (so it is not required to use `-v` when `-l` is used), prepended by several lines, a line per trace, with more detailed information about each trace.
 
-## Environment requirements
+The printout of `test_cdpa_attack.py` includes three lines:
+
+```bash
```

Comment:
`bash` keyword is used to highlight a shell commands sequence. In this case we have just some message printed to stdout, so it's better to use simple text highlighting.
```suggestion
```text
```